st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -88,7 +91,19 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        result = {
```

Comment:
Yes, and we'll be shipping the new version of `hubot-stackstorm` at the same time this PR is released. Hubot, on the other hand, will actually be compatible with both new and old versions (they're pretty easy to separate because they send different HTTP status codes).